### DISCUSSION TITLE
Complete backup

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#556] [Q&A] [answered]
There is one thing that I think has been discussed before but then not pursued any further: Creating a complete backup of Cruiser including all favourites of waypoints tracks and routes. 
This would be perfect as a complete data backup and to share a complete Cruiser setup including all data with friends. The inclusion of brouter data and maps could be made selectable due to the file size.
I would like something like this because it is easy to share the cruiser settings when preparing for a tour but very time-consuming to make the data and settings of "my collection" the same on several devices.
Such an option would also greatly support my efforts to promote Cruiser among my riding buddies.

**Comment by devemux86** [comment, marked as answer]
@GuG1050 Thanks for the suggestion. This is already discussed in:
- https://github.com/devemux86/cruiser/discussions/528

Profiles and favorites can be exported / imported as a whole by selecting "All".
Transferring maps is complicated due to their size, it's easier to download them.

**Reply by GuG1050** [reply]
That could be a good solution.  Would be interesting what other users think.

**Reply by bios4** [reply]
This was mentioned/discussed [just recently](https://github.com/devemux86/cruiser/discussions/545#discussioncomment-9530182), and also in an already existing (now closed?) [thread](https://github.com/devemux86/cruiser/discussions/528).
So: I fully support the suggestion! 👍

**Reply by mkrezdo** [reply]
same here - would be greate to share it with diffrent phones
